app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,25 @@
+class UserMailer < Devise::Mailer
+  helper :application # gives access to all helpers defined within `application_helper`.
+  helper :events
+  include Devise::Controllers::UrlHelpers # Optional. eg. `confirmation_url`
+  default from: "perlmutterteam@gmail.com"
```

Comment:
Should we have this set in the ENV file?

Reply:
Yes, adjusted